src/app/public/modules/overlay/overlay.component.ts

```diff
@@ -48,6 +52,8 @@ import {
   SkyOverlayContext
 } from './overlay-context';
 
+let uniqueZIndex = 1001; // Omnibar is 1000
```

Comment:
Should this be called `baselineIndex` or `startingIndex`? I'm not really sure what "unique" means unless I had prior knowledge of the omnibar.

Reply:
I can rename it, sure. The idea I was trying to communicate was that every overlay would get a unique, auto-incrementing z-index.

Reply:
Disregard.